.github/workflows/ci.yaml

```diff
@@ -96,7 +96,11 @@ jobs:
           # We don't have that right now, because we're not sure this pain point exists.
           uv-dependency-install-flags: "--all-extras --group tests"
       - name: Run tests
+        env:
+          ECMWF_DATASTORES_URL: ${{ secrets.ECMWF_DATASTORES_URL }}
+          ECMWF_DATASTORES_KEY: ${{ secrets.ECMWF_DATASTORES_KEY }}
         run: |
```

Comment:
```suggestion
```

Given you're not running the download tests, we don't need this here do we? (Same for similar use below)